src/com/palmergames/bukkit/towny/command/PlotCommand.java

```diff
@@ -1409,9 +1409,10 @@ public static void plotTestOwner(Resident resident, TownBlock townBlock) throws
 
 		} else {
 
+			final boolean asMayorInUnowned = player.hasPermission(PermissionNodes.TOWNY_COMMAND_PLOT_ASMAYORINUNOWNED.getNode());
 			boolean isSameTown = townBlock.getTownOrNull().hasResident(resident);
 
-			if (isSameTown && !isMayor)
+			if (isSameTown && (!isMayor || !asMayorInUnowned))

```

Comment:
This will prevent mayors using unowned plots like they would expect because the asMayorInUnowned will always be false. 

This can be fixed by adding the new node as a child node of towny.command.plot.* in the plugin.yml.